### PR TITLE
Implement 12 column grid in footer

### PR DESCRIPTION
**Pull request by robinandeer**
## Describe your changes

Implement the 12 column grid from Figma in the footer block.

## Justify why they are needed

Let's experiment with this in more places and maybe come up with a reusable component/styles.

## Checklist before requesting a review

- [ ] I have performed a self-review of my code

### CODE REVIEW

**Review by alebedev — APPROVED** [review]
Looks good!

File: apps/store/src/blocks/FooterBlock.tsx [hunk 1]
```diff
@@ -93,95 +93,86 @@ export const FooterBlock = ({ blok, onLocaleChange }: FooterBlockProps) => {
...
-  marginInline: 'auto',
+const Column = styled.div({
+  gridColumn: 'span 6',
 
-  [mq.md]: {
-    gridTemplateColumns: 'repeat(4, 1fr)',
+  [mq.md]: { gridColumn: 'span 3' },
+  [mq.xxl]: {
+    gridColumn: 'span 2',
+    ':first-child': { gridColumn: '3 / span 2' },
```
Comment (gustaveen) [inline apps/store/src/blocks/FooterBlock.tsx:147]: nit: same result, but played around a bit before and I think it's slightly easier to understand when you just override `gridColumnStart` 
```suggestion
    gridColumn: 'auto / span 2',
    ':first-child': { gridColumnStart: '3' },
```

**Review by gustaveen — APPROVED** [review]
Awesome!! 👌 Looks great to use the column layout for the footer 🙌

**Comment by robinandeer** [comment]
⏳ This pull request is set to merge as part of a Graphite merge job
Stack job ID: oJ5mYhvm2O6m3IUaNXZe.
See details on [graphite.dev](https://app.graphite.dev/github/pr/HedvigInsurance/racoon/1456)

**Pull request merged**